### PR TITLE
#815 improved issue & Pull Request templates

### PR DESCRIPTION
**Pull request by nioperas06**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Bug.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+about: Something is broken? 🔨
+---
+
+### Bug Report
+
+<!-- Fill in the relevant information below to help triage your issue. -->
+
+|    Q        |   A
+|------------ | ------
+| BC Break    | yes/no
```
Comment (PascalPrecht) [inline .github/ISSUE_TEMPLATE/Bug.md]: What does "BC Break" mean?

File: .github/ISSUE_TEMPLATE/Bug.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+| Version     | x.y.z
```
Comment (PascalPrecht) [inline .github/ISSUE_TEMPLATE/Bug.md]: Can we add a hint here how people can get the version number? For example, if it's the client app, they can find it as an attribute on the `<body>` element.

File: .github/ISSUE_TEMPLATE/Bug.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+#### Summary
+
+<!-- Provide a summary desciribing the problem you are experiencing. -->
```
Comment (PascalPrecht) [inline .github/ISSUE_TEMPLATE/Bug.md]: desciribing =? describing

File: .github/ISSUE_TEMPLATE/Proposal.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+name: 🤔 Proposal
+about: Have a nice proposal?
+---
+
+<!-- Fill in the relevant information below to help triage your issue. -->
+
+|    Q        |   A
+|------------ | -----
+| Version     | x.y.z
```
Comment (PascalPrecht) [inline .github/ISSUE_TEMPLATE/Proposal.md]: I don't think the version is relevant here, is it?

File: .github/ISSUE_TEMPLATE/Bug.md (hunk 3 above)
Comment (nioperas06) [reply .github/ISSUE_TEMPLATE/Bug.md]: Typo :sweat_smile:

File: .github/ISSUE_TEMPLATE/Bug.md (hunk 2 above)
Comment (nioperas06) [reply .github/ISSUE_TEMPLATE/Bug.md]: Sure! I thought the version should be the one inserted in client/server/cli/... package.json. Maybe I'm wrong. Where is this <body> element?

Comment (PascalPrecht) [reply .github/ISSUE_TEMPLATE/Bug.md]: Yes it's also in the `package.json` of every package inside `machinelabs/machinelabs`. But user may not actually see that. They just use the service and realize there's a bug. 

In fact, I think unless the issue originates from a developer, there's (right now) only `cli` and `client` that can be of interest.

File: .github/ISSUE_TEMPLATE/Proposal.md (hunk 4 above)
Comment (nioperas06) [reply .github/ISSUE_TEMPLATE/Proposal.md]: Yes, you're right!

File: .github/ISSUE_TEMPLATE/Bug.md (hunk 1 above)
Comment (nioperas06) [reply .github/ISSUE_TEMPLATE/Bug.md]: This term could be useful during libraries break changes; Obviously if we take a user point of view, it does not really make sense ... useful. 😕

Comment (nioperas06) [reply .github/ISSUE_TEMPLATE/Bug.md]: Okay 👍

**Comment by PascalPrecht** [comment]
@nioperas06 awesome thank you! Now we only need to do one last thing:

Squashing all those commits into a single one, that introduces this change.

Do you need help with that?

File: .github/ISSUE_TEMPLATE/Feature_Request.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+### Feature Request
+
+<!-- Fill in the relevant information below to help triage your issue. -->
+
+|    Q        |   A
+|------------ | ------
+| New Feature | yes
+| BC Break    | yes/no
```
Comment (SamVerschueren) [inline .github/ISSUE_TEMPLATE/Feature_Request.md]: `Breaking change`

File: .github/PULL_REQUEST_TEMPLATE/Improvement.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+### Improvement
+
+<!-- Fill in the relevant information below to help triage your issue. -->
+
+|    Q        |   A
+|------------ | ------
+| New Feature | yes
+| BC Break    | yes/no
```
Comment (SamVerschueren) [inline .github/PULL_REQUEST_TEMPLATE/Improvement.md]: `Breaking change`

File: .github/PULL_REQUEST_TEMPLATE/New_Feature.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+-->
+
+### New Feature
+
+<!-- Fill in the relevant information below to help triage your issue. -->
+
+|    Q        |   A
+|------------ | ------
+| New Feature | yes
+| BC Break    | yes/no
```
Comment (SamVerschueren) [inline .github/PULL_REQUEST_TEMPLATE/New_Feature.md]: `Breaking change`

File: .github/ISSUE_TEMPLATE/Bug.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+---
+
+### Bug Report
+
+<!-- Fill in the relevant information below to help triage your issue. -->
+
+|    Q               |   A
+|--------------------| ------
+| Breaking change    | yes/no
+| Version            | x.y.z
```
Comment (SamVerschueren) [inline .github/ISSUE_TEMPLATE/Bug.md]: I saw @PascalPrecht talking about a hint where they can find this version? Should we add something like this in the comment above?

> Fill in the relevant information below to help triage your issue. You can find the client version when inspecting the body or you can find it in the relevant package.json file.

**Comment by PascalPrecht** [comment]
@nioperas06 it seems something when wrong in your PR with your commit history. I went ahead, took your commits, squashed them together and pushed them into `master` as this PR has been reviewed.

This landed as: https://github.com/machinelabs/machinelabs/commit/5d1c72823d60b8560931b5352f5d237b2307c559

**Pull request closed without merging**